src/components/frontend-engine/types.ts

```diff
@@ -45,6 +45,7 @@ export interface IFrontendEngineProps<V = undefined> {
 	onChange?: ((values: TFrontendEngineValues, isValid?: boolean | undefined) => unknown) | undefined;
 	/** Submit event handler, will receive the form data if form validation is successful */
 	onSubmit?: (values: TFrontendEngineValues) => unknown | undefined;
+	onSubmitError?: (errors: TFrontendEngineValues) => unknown | undefined;
```

Comment:
to add a jsdoc comment to describe this property

Reply:
done